STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/_az_ulib_dm_blob_ustream_interface.h

```diff
@@ -0,0 +1,52 @@
+#ifndef AZ_ULIB_DM_BLOB_USTREAM_INTERFACE_H
+#define AZ_ULIB_DM_BLOB_USTREAM_INTERFACE_H
+
+#include "az_ulib_result.h"
+#include "az_ulib_ustream.h"
+#include "azure/az_core.h"
+#include "nx_api.h"
+#include "nx_web_http_client.h"
+#include "nx_wifi.h"
+#include "nxd_dns.h"
+#include "stm_networking.h"
+#include "wifi.h"
+
+
+#ifndef __cplusplus
+#include <stdint.h>
+#else
+#include <cstdint>
+#endif /* __cplusplus */
+
+#define DCF_WAIT_TIME 600
+
+#include "azure/core/_az_cfg_prefix.h"
+
+typedef struct az_blob_http_cb_tag
+{
+  struct
+  {
+    NX_WEB_HTTP_CLIENT http_client;
+    NX_PACKET* packet_ptr;
+    NXD_ADDRESS* ip;
+    CHAR* resource;
+    CHAR* host;
+    bool last_chunk;
+  } _internal;
+} az_blob_http_cb;
+
+AZ_NODISCARD az_result create_ustream_from_blob(
```

Comment:
>create_ustream_from_blob [](http://example.com/codeflow?start=23&length=24)

Please identify as az API, somehing like `az_blob_create_ustream_from_blob()`

Reply:
If your intention is make this API public, the file name should not start with `_`. If it is not public, function and structures should start with `_` and it is not necessary to create the nest `_intrenal` struct.